src/style.css

```
@@ -0,0 +1,138 @@
+html {
+  margin: 0;
+}
+
+body {
+  margin: 0;
+}
+
+.conteiner {
+  position: relative;
+}
+
+ul {
+  margin-top: 0;
+  margin-bottom: 0;
+  padding: 0;
+}
+
+li {
+  list-style: none;
+  display: flex;
+}
+
+.stars--0 {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+.stars--1 {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+.stars--2 {
+  display: flex;
+}
+
+.stars--3 {
+  display: flex;
+}
+
+.stars--4 {
+  display: flex;
+}
+
+.stars--5 {
+  display: flex;
+}
```

Do not repeat your self
```suggestion
.stars {
  display: flex;
}
```